tests/frontend/test_kibana_app/cypress/integration/utils/driver.js

```diff
@@ -63,18 +63,46 @@ export const clearSession = () => {
   cy.clearCookies();
 };
 
-export const setCookies = (cookieObj) => {
+export const setCookies = (cookieFromFile) => {
   try {
-  cookieObj.forEach((element) => {
-    cy.setCookie(element.name, element.value);
-  });
-} catch (e) {
-}
+    cy.getCookies().then((currentCookie) => {
+      if (currentCookie.length != 0) {
+        cookieFromFile.forEach((element) => {
+          cy.setCookie(element.name, element.value);
+        });
+      }else{
+        cy.readFile('cookie.json').then((cookieFile) => {
+          cy.log('cookie',cookieFile);
+            cookieFile.forEach(element => { 
+            cy.setCookie(element.name, element.value);
+          })
+        })
+      }
+    });
+  } catch (e) {
+  }
 }
 
+// export const checkCookie = () => {
```

Comment:
do we need these commented lines? or are old deprecated methods?
if we don't use them please delete them.

Reply:
done